### PR TITLE
improve export analyzer

### PR DESCRIPTION
**Pull request by mikehelmick**
* return non zero status code on error in file
* make time ranges flag configurable
* add flag to not print JSON dump

**Release Note**

```release-note
NONE
```

### CODE REVIEW

File: tools/export-analyzer/main.go [hunk 1]
```diff
@@ -18,26 +18,37 @@ package main
...
 	flag.Parse()
-
 	if *filePath == "" {
 		log.Fatal("--file is required.")
 	}
+	if *allowedTEKAge < time.Duration(0) {
+		log.Fatalf("--tekage must be a positive duration, got: %v", *allowedTEKAge)
+	}
+	if *symptomDayLmit < 0 {
+		log.Fatalf("--sypmtomdays must be >=0, got: %v", *symptomDayLmit)
```
Comment (sethvargo) [inline tools/export-analyzer/main.go:50]:
```suggestion
		log.Fatalf("--sypmtomdays must be >=0, got: %v", *symptomDayLmit)
```

File: tools/export-analyzer/main.go [hunk 2]
```diff
@@ -18,26 +18,37 @@ package main
...
+	"github.com/hashicorp/go-multierror"
 )
 
 var (
-	filePath = flag.String("file", "", "Path to the export zip file.")
-	quiet    = flag.Bool("q", false, "run in quiet mode")
+	filePath       = flag.String("file", "", "Path to the export zip file.")
+	printJSON      = flag.Bool("json", true, "Print a JSON representation of the output")
+	quiet          = flag.Bool("q", false, "run in quiet mode")
+	allowedTEKAge  = flag.Duration("tekage", 14*24*time.Hour, "max TEK age in checks")
```
Comment (sethvargo) [inline tools/export-analyzer/main.go:37]: Can we do tek-age and symptom-days for clarity?

File: tools/export-analyzer/main.go [hunk 3]
```diff
@@ -50,43 +61,63 @@ func main() {
...
 
-	prettyJSON, err := json.MarshalIndent(keyExport, "", "  ")
-	if err != nil {
-		log.Fatalf("error pretty printing export: %v", err)
+	if *printJSON {
+		prettyJSON, err := json.MarshalIndent(keyExport, "", "  ")
+		if err != nil {
+			log.Fatalf("error pretty printing export: %v", err)
+		}
+		log.Printf("%v", string(prettyJSON))
```
Comment (sethvargo) [inline tools/export-analyzer/main.go:77]: If this is intended to be here, can we prefix with "output" or something. If it's debug, let's remove it.

File: tools/export-analyzer/main.go [hunk 4]
```diff
@@ -18,26 +18,37 @@ package main
...
 	"github.com/google/exposure-notifications-server/internal/export"
 	exportpb "github.com/google/exposure-notifications-server/internal/pb/export"
 	"github.com/google/exposure-notifications-server/internal/publish/model"
+	"github.com/hashicorp/go-multierror"
 )
 
 var (
-	filePath = flag.String("file", "", "Path to the export zip file.")
-	quiet    = flag.Bool("q", false, "run in quiet mode")
+	filePath       = flag.String("file", "", "Path to the export zip file.")
```
Comment (whaught) [inline tools/export-analyzer/main.go:34]:
```suggestion
	filePath       = flag.String("file", "", "path to the export zip file.")
```

File: tools/export-analyzer/main.go [hunk 5]
```diff
@@ -18,26 +18,37 @@ package main
...
+	printJSON      = flag.Bool("json", true, "Print a JSON representation of the output")
```
Comment (whaught) [inline tools/export-analyzer/main.go:35]:
```suggestion
	printJSON      = flag.Bool("json", true, "print a JSON representation of the output")
```

File: tools/export-analyzer/main.go (hunk 3 above)
Comment (mikehelmick) [reply tools/export-analyzer/main.go:77]: yes, that was the original reason for this tool to print the export file contents

**Pull request merged**